eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Generator/Trace/Chain.hs

```diff
@@ -241,11 +242,11 @@ registerGenesisStaking
       newDState :: DState (Crypto era)
       newDState =
         (_dstate oldDPState)
-          { _rewards =
-              Map.map (const $ Coin 0)
-                . Map.mapKeys KeyHashObj
-                $ sgsStake,
-            _delegations = Map.mapKeys KeyHashObj sgsStake
+          { _unified =
+              UM.unify
+                (Map.map (const $ Coin 0) . Map.mapKeys KeyHashObj $ sgsStake)
+                (Map.mapKeys KeyHashObj sgsStake)
+                (Map.empty)
```

Comment:
I wonder if the (Map.empty) should be the Ptrs equivalent of the oldDPState, we should probably fix this.